### PR TITLE
Harvardinformatics bam2wig

### PR DESCRIPTION
**Pull request by hmehlan**
This PR replaces #153 ; 
rather than removing bam2wig completely (which is not desirable for all use cases, as it's more efficient than bamToWig.py), this PR provides modifications that make it easier to build; specifically, only htslib (>= 1.10.0) is required rather than also requiring (source builds of) bcftools and samtools.
ubuntu 20.04 and debian 11 alternatively provide the package libhts-dev.

### CODE REVIEW

File: tests/auxprogs/bam2wig/test_bam2wig.py [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+from pathlib import Path
+import os
+import sys
+import shutil
+
+if not (os.path.exists('tests')):
+    sys.exit('Wrong working directory!\nTest files must be accessible in ./tests folder.')
+sys.path.insert(0, '.') # make modul tests.utils accessible
+
+from tests.utils import aug_assertions
```
Comment (MarioStanke) [inline tests/auxprogs/bam2wig/test_bam2wig.py:13]:
```    
from tests.utils import aug_assertions
ModuleNotFoundError: No module named 'tests.utils'
```

File: tests/utils/aug_argparse.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    parser.add_argument('--compare',
+                        action='store_true',
+                        help='Compare generated results with reference results.')
+    parser.add_argument('--html',
+                        action='store_true',
+                        help='Save diff results in html file.')
+    parser.add_argument('--clean',
+                        action='store_true',
+                        help='Remove all files created during the tests.')
+    return parser
```
Comment (MarioStanke) [inline tests/utils/aug_argparse.py:16]:
```suggestion
    return parser
    
```

**Review by MarioStanke — CHANGES_REQUESTED** [review]
make clean requires Python now. I suspect some dependencies may not be necessary to delete files and the dependency can be dropped.
In any case,  the required python module for testing does not appear to be mentioned as a requirement to run tests. 


```
# make clean
tests/auxprogs/bam2wig/test_bam2wig.py --clean
Traceback (most recent call last):
  File "tests/auxprogs/bam2wig/test_bam2wig.py", line 13, in <module>
    from tests.utils import aug_assertions
ModuleNotFoundError: No module named 'tests.utils'

```

no newline at end of aug_assertions.py, aug_path.py, aug_process.py.

File: tests/auxprogs/bam2wig/test_bam2wig.py (hunk 1 above)
Comment (MarioStanke) [reply tests/auxprogs/bam2wig/test_bam2wig.py:13]: I ran `make clean` in the top-level dir.

File: tests/utils/aug_argparse.py (hunk 2 above)
Comment (hmehlan) [reply tests/utils/aug_argparse.py]: newline added

**Review by MarioStanke — APPROVED** [review]

**Pull request merged**